README.md

```diff
@@ -1,3 +1,129 @@
-# Webex Components
+<h1 align='center' style='border-bottom: none;'>Webex Components</h1>
+<h3 align='center'>Library of React components to easily embed into your web applications!</h3>
+<p align='center'>
+<a href='https://circleci.com/gh/webex/components'>
+    <img alt='CircleCI' src='https://circleci.com/gh/webex/components.svg?style=shield'>
+  </a>
+  <a href='#badge'>
+    <img alt='semantic-release' src='https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg'>
+  </a>
+  <a href='https://www.npmjs.com/package/@webex/components'>
+    <img alt='npm latest version' src='https://img.shields.io/npm/v/@webex/components?label=npm%40latest'>
+  </a>
+  <a href='https://github.com/webex/components/blob/master/package.json#L28'>
+    <img src='https://img.shields.io/npm/l/webex.svg' alt='license'>
+  </a>
+</p>
 
-The library for Webex Components.
+**Webex Components** is a set of *React* components following Webex Teams standard styling, aimed at react developers that want to embed the components into their applications.
+
+
+## Table of Contents
+- [Install](#install)
+- [Usage](#usage)
+- [Contributions](#contributions)
+- [License and Support](#license)
+
+<a name='install'></a>
+## Install
+
+```bash
+npm i -S @webex/components @momentum-ui/react
+```
+
+<a name='usage'></a>
+## Usage
+**Webex Components** are using *Cisco style standard* presentational  [momentum ui](https://github.com/momentum-design/momentum-ui) react components. And because of that we need to import all the fonts, icons, images and core *CSS* manually. There are two ways to do this:
+
+### 1. CSS
+1. Copy the '@momentum-ui/core/fonts' directory to the 'fonts' directory for your site.
+2. Copy the '@momentum-ui/core/images' directory to the 'images' directory for your site.
+3. Copy the '@momentum-ui/core/css/momentum-ui.min.css' to your styles/css directory.
+4. In the `<head>` of your HTML reference the location of your momentum-ui.min.css.
+
+    `<link rel='stylesheet' href='path/to/momentum-ui/css/momentum-ui.min.css'>`
+
+***
+### 2. SCSS
+1. Copy the '@momentum-ui/icons/fonts' directory to the 'fonts' directory for your site.
+2. Add an '$brand-font-folder, $icon-font-path, and $images-path' variables to your app's variables.scss file.
+
+    ```scss
+    $brand-font-folder: 'path/to/fonts/directory';
+    $icon-font-path: 'path/to/fonts/directory';
+    $images-path: 'path/to/images/directory';
+    ```
+
+3. Import '@momentum-ui/core/scss/momentum-ui.scss' into your main entry Scss file AFTER the variables.
+
+    `@import '~@momentum-ui/core/scss/momentum-ui';`
+
+4. Compile your Scss using your static compiler or bundler (Webpack, Grunt, Gulp, etc.)
+***
+
+Now that we have the targeted style library imported into our application, we can go ahead and import the components as it shows in the following code:
```

Comment:
The first part of the sentence is a bit confusing, you can rephrase as:

`
Once Momentum styles have been imported into your project, you can start using the components. For example:
`